src/ProjectFileManager.cpp

```diff
@@ -165,20 +167,32 @@ auto ProjectFileManager::ReadProjectFile(
    const bool bParseSuccess = parseResult.has_value();
    
    bool err = false;
+   bool waveTrackLinkTypeChanged = false;
 
    TranslatableString otherError;
 
    if (bParseSuccess)
    {
       auto &tracks = TrackList::Get(project);
       for (auto t : tracks) {
+         const auto linkType = t->GetLinkType();
          // Note, the next function may have an important upgrading side effect,
          // and return no error; or it may find a real error and repair it, but
          // that repaired track won't be used because opening will fail.
          if (!t->LinkConsistencyFix()) {
             otherError = XO("A channel of a stereo track was missing.");
             err = true;
          }
+         if(!err && linkType != t->GetLinkType())
+         {
+            //Not an elegant way to deal with wave track linking
+            //compatibility between versions
+            if(auto waveTrack = dynamic_cast<WaveTrack*>(t))
+            {
+               RealtimeEffectList::Get(*waveTrack).Clear();
```

Comment:
I think this is incomplete.

Because `SetLinkType` with argument `None` caused duplication of the effect stack into what was the right channel before it returned.

But this erases the effect stack only on one of the resulting mono tracks.